src/Core/Migration/FluentSwitch.cs

```diff
@@ -0,0 +1,31 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+public partial class FluentSwitch
+{
+    /// <summary>
+    /// Gets or sets the content to be rendered inside the component.
+    /// </summary>
+    [Parameter]
+    [Obsolete("This property is not supported anymore and will be removed in a future release.")]
+    public RenderFragment? ChildContent { get; set; }
+
+    /// <summary>
+    /// Gets or sets the checked message
+    /// </summary>
+    [Parameter]
+    //[Obsolete("This property is not supported anymore and will be removed in a future release.")]
```

Comment:
Can you uncomment these `Obsolete` attribute?

Reply:
If there is an `[Obsolete]`, I doesn't compile anymore, with this message:

`'FluentSwitch.UncheckedMessage' is obsolete: 'This property is not supported anymore and will be removed in a future release.'`

Reply:
You need to add a `pragma` condition. We already used this in the code.